build/root/WORKSPACE

```diff
@@ -29,6 +29,14 @@ http_archive(
     urls = mirror("https://github.com/coreos/etcd/releases/download/v%s/etcd-v%s-linux-amd64.tar.gz" % (ETCD_VERSION, ETCD_VERSION)),
 )
 
+http_archive(
+    name = "com_coreos_etcd_arm64",
```

Comment:
maybe also change the `com_coreos_etcd` dependency for amd64 to `com_coreos_etcd_amd64` for consistency?

Reply:
Done.